components/form/builder/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@s-ui/form-builder",
```

Comment:
To follow the same package name we're using, this should be `@s-ui/react-form-builder`